redwood-widget-compose/src/commonMain/kotlin/app/cash/redwood/widget/compose/ComposeWidgetChildren.kt

```diff
@@ -17,22 +17,22 @@ package app.cash.redwood.widget.compose
 
 import androidx.compose.runtime.Composable
 import androidx.compose.runtime.mutableStateListOf
-import app.cash.redwood.widget.MutableListChildren
+import app.cash.redwood.LayoutModifier
 import app.cash.redwood.widget.Widget
 
 public class ComposeWidgetChildren : Widget.Children<@Composable () -> Unit> {
-  private val _widgets = MutableListChildren<@Composable () -> Unit>(mutableStateListOf())
-  public val widgets: List<Widget<@Composable () -> Unit>> get() = _widgets
+  private val _widgets = mutableStateListOf<Pair<Widget<@Composable () -> Unit>, LayoutModifier>>()
+  public val widgets: List<Pair<Widget<@Composable () -> Unit>, LayoutModifier>> get() = _widgets
```

Comment:
Hmm why the `Pair`? Aren't the `LayoutModifier` instances available here the same as those you can get to from each `Widget` instance?

Reply:
They are, but we need a way to trigger recomposition of an item if its layout modifiers change.

Reply:
Ah, yeah. Should that be a separate list with dummy reads in the composable? I guess the real version of my original question is: do we need to expose the pairs publicly to the parents?